fluentd/tests/test_integration.py

```diff
@@ -14,7 +14,7 @@
 @pytest.mark.usefixtures("dd_environment")
 def test_fluentd_exception(aggregator, check):
     instance = {"monitor_agent_url": BAD_URL, "plugin_ids": ["plg2"], "tags": ["test"]}
-
+    check = check({}, instance)
```

Comment:
(minor) IMHO could be easier to read if we inline check creation, instead of using a pytest fixture.

```
check = Fluentd(CHECK_NAME,{}, [instance])
```